flight/mcu/lib/teensyShared/ActiveControl.h

```diff
@@ -1,41 +1,37 @@
 #include <PWMServo.h>
-#include <math.h>
 #include <rk4.h>
 
-#include <array>
-
 #include "ServoControl.h"
-#include "dataLog.h"
 
-using std::array;
+class Controller;
+
+extern Controller activeController;
 
 class Controller {
    public:
-    void ctrlTickFunction(pointers* pointer_struct);
+    void ctrlTickFunction();
     bool ActiveControl_ON();
-    Controller(struct pointers* pointer_struct, PWMServo* controller_servo);
+    Controller();
+    void init();
+
+    void startup();
 
     void setLaunchPadElevation();
 
-    PWMServo* controller_servo_;
-    mutex_t* dataMutex_state_;
-    stateData* stateData_;
+    PWMServo controller_servo_;
     rk4 rk4_;
     float kp = 0.00008;
 
-    float min_extension = -2.0 / 1000.0;
+    float min_extension = 0.0;
```

Comment:
Why change the min extension?

Reply:
That wasn't me, I just propagated the change from something was merged in.